geoips/schema/__init__.py

```diff
@@ -1 +1,2 @@
+"""Schema init file."""
```

Comment:
@jsolbrig I additionally made some updates to get flake8 and doc build to pass (adding docstrings, etc)

Reply:
Great, thanks for doing that!